girder_nlisim/models.py

```diff
@@ -47,6 +47,27 @@ def setSimulationComplete(self, simulation):
         simulation.get('nli', {})['complete'] = True
         return self.save(simulation)
 
+    def get_summary_stats(self, simulation, user) -> Dict[str, Dict]:
+        """Creates the summary statistics of a simulation in json form."""
+        # I'm just going to assume that all subfolders are for time-steps but I'll skip them
+        # if they don't have a time field set. (or, horrors, if it is negative)
```

Comment:
This could be done as a custom mongo query on `parentId` and `meta.time`.  Though, it's not likely to matter much in practice.

Reply:
That sounds like a good idea, I'll put in the todo list.